coins/coins.go

```diff
@@ -283,71 +283,68 @@ func (t *ERC20TokenInfo) SanitizedSymbol() string {
 
 // ERC20TokenAmount represents some amount of an ERC20 token in the smallest denomination
 type ERC20TokenAmount struct {
-	Amount    *apd.Decimal    `json:"amount" validate:"required"` // in smallest non-divisible units of token
+	Amount    *apd.Decimal    `json:"amount" validate:"required"` // in standard units
```

Comment:
I was getting confused by this, and I don't think I was the only one. In other places where we represent non-standard units using `*apd.Decimal`, we are using a type alias. This was the only place where we were declaring the field as `*apd.Decimal` and it was not in standard units. And the name of the field didn't clarify it either. I considered creating a new type alias or renaming the field, but just storing the value in standard units seemed cleaner.